docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3"
+
+services:
+  mysql:
+    image: mysql
+    environment:
+      MYSQL_USER: ${MYSQL_USER}
+      MYSQL_PASSWORD: ${MYSQL_PASSWORD}
+      MYSQL_DATABASE: ${MYSQL_DATABASE}
```

Comment:
didn't know u could use env variables here - this is really cool

Reply:
yeah, and the mysql image from docker hub automatically generates the user and database and privileges and so on, which makes things nice and easy for the db init :D